### PR TITLE
Added reset line offset as parameter

### PR DESCRIPTION
**Pull request by ovidiug1980**
The gpio line offset is configurable now and should be passed from the modemfwd helpers manifest.

### CODE REVIEW

File: ql-modem-helper.c [hunk 1]
```diff
@@ -184,7 +185,8 @@ static int print_help(int argc)
         fprintf(stderr,"   --%s\n", kGetFirmwareInfo);
         fprintf(stderr,"   --%s\n", kPrepareToFlash);
         fprintf(stderr,"   --%s\n", kFlashFirmware);
-	fprintf(stderr,"   --%s\n", kFlashModeCheck);
+				fprintf(stderr,"   --%s\n", kResetGpioLine);
```
Comment (naveenquectel) [inline ql-modem-helper.c]: Please correct the indentation.

File: ql-modem-helper.c [hunk 2]
```diff
@@ -300,11 +304,11 @@ int main(int argc, char *argv[])
 						get_version();
                 return 0;
             case 'P':
-		if (mbim_prepare_to_flash()) {
-		       return EXIT_FAILURE;
-		}
+								if (mbim_prepare_to_flash()) {
```
Comment (naveenquectel) [inline ql-modem-helper.c]: Please correct the indentation.

**Review by naveenquectel — COMMENTED** [review]
Please correct the indentation and submit again.

File: ql-modem-helper.c (hunk 2 above)
Comment (ovidiug1980) [reply ql-modem-helper.c]: Ok done.

**Review by naveenquectel — APPROVED** [review]
Looks good to me.

**Pull request merged**